mne/viz/_3d.py

```diff
@@ -415,6 +453,20 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         View to use. See surfer.Brain().
     colorbar : bool
         If True, display colorbar on scene.
+    clim : str | dict
+        Colorbar properties specification. If 'auto', set clim automatically
+        based on data percentiles. If dict, should contain:
+            kind : str
+                Flag to specify type of limits. 'value' or 'percent'.
+            lims : length 3 list, array, or tuple
+                Note: Only use this if 'colormap' is not 'mne_analyze'.
+                Left, middle, and right bound for colormap.
+            pos_lims : length 3 list, array, or tuple
+                Note: Only use this if 'colormap' is 'mne_analyze'.
+                Left, middle, and right bound for colormap. Positive values
+                will be mirrored directly across zero during colormap
+                construction to obtain negative control points.
```

Comment:
Not sure if I'm following. Ambiguous if they need to specify (5, 10, 15) instead of (-15, 0, 15)? I can change the docstring to further underscore the fact that it needs to positive control points. The old flims only had users specify 3 positive points for the colormap, so it's sticking with the old intuition.